app/js/app.js

```diff
@@ -17,12 +17,10 @@ angular.module('app', [
 ]).config(function(RestangularProvider) {
 	RestangularProvider.setBaseUrl('https://api.tnyu.org/v2');
 
-	var ApiKey = "";

```

Comment:
@ethanresnick That doesn't work. Abhi made it blank a while ago because it was breaking stuff. I got rid of the header that it was corresponding too also.